.pylintrc

```diff
@@ -50,7 +50,21 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=print-statement,parameter-unpacking,unpacking-in-except,old-raise-syntax,backtick,long-suffix,old-ne-operator,old-octal-literal,import-star-module-level,raw-checker-failed,bad-inline-option,locally-disabled,locally-enabled,file-ignored,suppressed-message,useless-suppression,deprecated-pragma,apply-builtin,basestring-builtin,buffer-builtin,cmp-builtin,coerce-builtin,execfile-builtin,file-builtin,long-builtin,raw_input-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,no-absolute-import,old-division,dict-iter-method,dict-view-method,next-method-called,metaclass-assignment,indexing-exception,raising-string,reload-builtin,oct-method,hex-method,nonzero-method,cmp-method,input-builtin,round-builtin,intern-builtin,unichr-builtin,map-builtin-not-iterating,zip-builtin-not-iterating,range-builtin-not-iterating,filter-builtin-not-iterating,using-cmp-argument,eq-without-hash,div-method,idiv-method,rdiv-method,exception-message-attribute,invalid-str-codec,sys-max-int,bad-python3-import,deprecated-string-function,deprecated-str-translate-call,C,R
+disable=
```

Comment:
Removed all disabled warnings that was not violated and made all errors explicit instead of disabling entire categories, like `C`. In addition I did split it over multiple lines to make it easier and less verbose to remove one of them.

Reply:
This is painful to review, but I guess it's easier to discuss ehther to disable a warning or not the first time it pops up again

Reply:
This is the minimal list of disables that let the code pass the linter. Some of them should indeed be removed, but now you can remove a single line, fix the problems and create a (tiny) PR that resolves it. The other thing that happened was that I removed all disabling of categories (R and C) and made them all explicit.